### DISCUSSION TITLE
Loja de Aplicativos não carrega

### DISCUSSION THREAD

**Discussion by Chronos093** [IceWhaleTech/CasaOS#300] [Q&A] [answered]
>Boa tarde,
Eu instalei o CasaOs e estou tentando instalar os aplicativos da loja, mas quando coloco para abrir a loja fica carregando e não aparece o conteúdo. Poderia me ajudar.

![image](https://user-images.githubusercontent.com/70582239/174150604-4866e4c9-bbd6-4108-afef-d6c844857491.png)

**Comment by Lauren-ED209** [comment, marked as answer]
It's fixed. And please sumbit to issue part, next time. Thanks. ;)

**Reply by Chronos093** [reply]
Thank you.
